CONTRIBUTORS.md

```diff
@@ -0,0 +1,6 @@
+# Contributors
+
+| GitHub username | Discord username | Contributions | Additional Info |
+|-----------------|------------------|---------------|-----------------|
+
+| [mehta-avi](https://github.com/mehta-avi) | rbwings | Documentation |[My website](avimehta.info) |
```

Comment:
It doesn't seem to be rendering as a table.

Reply:
Fixed this. There was an additional line break which I removed.